src/components/Main.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+
+
+const Main = () =>{
+    return (<section className="main"></section>)
```

Comment:
Suggestion for when y’all go to merge: return a `Fragment` here with the `<Map />` component from @jendevelops’ & @ajiles91’s PR inside it

Reply:
OK, their PR is on `master` so please merge `master` into your branch and return `<Map/>` here. Thanks!

Reply:
I attempted to implement the suggestion with regards to the Fragment, but a Fragment doesn't appear to accept a className, which threw off the styling on main, i.e. the height specified. Hope this still works.